### ISSUE TITLE
Not working with jdk11

### ISSUE THREAD

**Issue by kalle77** [seahen/maven-s3-wagon#7]
When upgrading to jdk11 I get an error due to lack of jaxb-dependency and the add-module option as a WO has been removed. It seems that it is due to the aws-sdk the dependency is needed for wagon. I already have the jaxb-api in my pom but this dependency is not picked up by the plugins/extensions. I think just adding the jaxb-api in the maven-s3-wagon will remedy the issue, thanks.

**Comment by MassimoSporchia** [comment]
@kalle77 @seahen

**Comment by MassimoSporchia** [comment]
Published:
```
<groupId>com.github.massimosporchia</groupId>
<artifactId>maven-s3-wagon</artifactId>
<version>1.3.1-SNAPSHOT</version>
```

Please let me know whenever you merge, so that I can remove mine.
Kudos to @kalle77

**Comment by velo** [comment]
FWIW, I noticed that this ticket was waiting for to be merged and released for months, so I decided to cut merge it with my changes and release it

```
            <extension>
                <groupId>com.marvinformatics</groupId>
                <artifactId>maven-s3-wagon</artifactId>
                <version>1.3.2</version>
            </extension>
```
https://repo1.maven.org/maven2/com/marvinformatics/maven-s3-wagon/1.3.2/

Includes https://github.com/seahen/maven-s3-wagon/pull/6 and https://github.com/seahen/maven-s3-wagon/pull/10

**Comment by Flo354** [comment]
Thanks @velo !

**Comment by seahen** [comment]
Fixed in Release 1.3.1 ( https://mvnrepository.com/artifact/com.github.seahen/maven-s3-wagon/1.3.1 ) 
@MassimoSporchia sorry about the delay, I hadn't checked up in awhile.  @velo thanks for covering.
